docs/packages.md

```diff
@@ -11,6 +11,22 @@ All LinuxKit packages are:
 - Derived from well-known (and signed) sources for repeatable builds.
 - Build with multi-stage builds to minimise their size.
 
+## Package source
+
+A package source consists of a directory containing two files:
+
+- `build.yml`: contains metadata associated with the package
+- `Dockerfile`: contains the steps to build the package.
+
+`build.yml` contains the following fields:
+
+- `image` _(string)_: *(mandatory)* The name of the image to build
+- `org` _(string)_: The hub/registry organisation to which this package belongs
+- `arches` _(list of string)_: The architectures which this package should be built for (valid entries are `GOARCH` names)
+- `gitrepo` _(string)_: The git repository where the package source is kept.
```

Comment:
This is hardcoded to `https://github.com/linuxkit/linuxkit` in `pkg/package.mk` today because you can't find it by shelling out to git, even if you assume the relevant remote is named `origin` this may point to some private name (e.g. `git@github.com:linuxkit/linuxkit`) rather than the desired name (https://github.com/linuxkit/linuxkit).

I eventually intend to deal with this by being able find a `.build-defaults.yml` in the root of the git tree which can override this and other things, such as `org` with local defaults.

Reply:
Ah, good point